dev-docs/bidders/equativ.md

```diff
@@ -24,35 +33,67 @@ The Equativ bidder adapter requires setup and approval from the Equativ service
 ### Bid params
 
 {: .table .table-bordered .table-striped }
-
 | Name | Scope | Description | Example | Type |
 |------|-------|-------------|---------|------|
 | `networkId` | required | The network identifier you have been provided with. _See **Bid Parameter Usage** notes below for more information_. | `1234` | `integer` |
 | `siteId` | required | The placement site ID. _See **Bid Parameter Usage** notes below for more information_. |`1234` | `integer` |
 | `pageId` | required | The placement page ID. _See **Bid Parameter Usage** notes below for more information_. | `1234` | `integer` |
 | `formatId` | required | The placement format ID. _See **Bid Parameter Usage** notes below for more information_. | `1234` | `integer` |
 
-#### Bid Parameter Usage 
+#### Bid Parameter Usage
+
 Different combinations of parameters are required depending upon which ones you choose to use.
 
 There are three options for passing bidder parameters:
 
-- **Option 1**.  Specify `networkId` by itself (_without_ `siteId`, `pageId` and `formatId`), or
-- **Option 2**.  Specify `siteId` _and_ `pageId` _and_ `formatId` (all together) _without_ `networkId`, or
-- **Option 3**.  Specify _none_ of the above parameters, and instead use either  `ortb2.site.publisher.id` or `ortb2.app.publisher.id`
+- **Option 1**. Specify `networkId` by itself (_without_ `siteId`, `pageId` and `formatId`), or
+- **Option 2**. Specify `siteId` _and_ `pageId` _and_ `formatId` (all together) _without_ `networkId`, or
+- **Option 3**. Specify _none_ of the above parameters, and instead use either `ortb2.site.publisher.id` or `ortb2.app.publisher.id`
 
 See **Sample Banner Setup** for examples of these parameter options.
 
 ### Supported Media Types
 
 {: .table .table-bordered .table-striped }
-
 | Type | Support |
 |---|---|
 | `banner` | Supported |
 | `video` | Not currently supported |
 | `native` | Not currently supported |
 
+### First Party Data
+
+You can use the `ortb2` property for setting first-party data. The following use cases are supported:
+
+- `ortb2.site.publisher.id`
+- `ortb2.app.publisher.id`
+
+The `app` or `site` object properties can be used as a way to specify an ID for Equativ to use to identify your requests (through `publisher.id`), as shown with the two examples below:
```

Comment:
Setting site/app.publisher.id to the Equative id could be problematic if the pub uses a Prebid Server that's not yours.

I would recommend making this bidder-specific First Party Data:

```
pbjs.setBidderConfig({
   bidders: ["equative"],
   config: {
      ortb2: {
        site: {
          publisher: { id: 42 }
        }
      }
   }
});
```

FWIW, calling this first party data support is pretty weak, but I'll let others reject it.

Reply:
That's a good point; I agree that `setBidderConfig()` is probably the better option.

With that change, I'm not sure what FPD support might remain with the adapter (and so it might go from "yes" to "no").  I'll follow up with the exchange team to see what they would read.

Reply:
I went ahead and got rid of the First Party Data section for now (and indicated `false` on FPD support).  We will be making further changes to this doc as we build out the new adapter.  Thanks for the detailed feedback, it's been very helpful.